### PR TITLE
Package Maintenance update

### LINKED ISSUE #62: DeepEcho Package Maintenance Updates
_This is a placeholder issue for overall package maintenance updates for the next minor DeepEcho release: `0.4.0`_

### PR DESCRIPTION
**Pull request by pvk-developer**
Resolves #62

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,22 +1,106 @@
...
 
+                      A "Synthetic Data Creation Service" is a commercial offering
+                      that allows third parties (other than your employees and
+                      contractors) to access the functionality of the Licensed
+                      Work so that such third parties directly benefit from the
+                      data processing, machine learning or synthetic data creation
+                      features of the Licensed Work.
+
+Change Date:          Change date is four years from release date.
+Please see https://github.com/sdv-dev/DeepEcho/releases for exact dates.
```
Comment (npatki) [inline LICENSE]: Let's put some spaces at the beginning of this line. It the "Please" should start right underneath the sentence "Change date is four years ..."

**Review by npatki — CHANGES_REQUESTED** [review]
Small spacing change but the rest LGTM

File: conda/meta.yaml [hunk 2]
```diff
@@ -31,9 +31,8 @@ requirements:
 
 about:
   home: "https://github.com/sdv-dev/DeepEcho"
-  license: MIT
```
Comment (amontanez24) [inline conda/meta.yaml:34]: I think we can just delete this file

**Review by npatki — APPROVED** [review]
LGTM!

**Review by amontanez24 — APPROVED** [review]
:shipit:

**Pull request merged**